app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
```

Comment:
Are you adding anything to these files?
If not, please remove them

Reply:
Removed.